src/utils/ExtensionLoader.js

```diff
@@ -393,27 +394,54 @@ define(function (require, exports, module) {
-                defaultExtensions = [
-                    // Core extensions we want to support in the browser
-                    "CSSCodeHints",
-                    "HTMLCodeHints",
-                    "JavaScriptCodeHints",
-                    "SVGCodeHints",
-                    "HtmlEntityCodeHints",
-                    "InlineColorEditor",
-                    "JavaScriptQuickEdit",
-                    "QuickOpenCSS",
-                    "QuickOpenHTML",
-                    "QuickOpenJavaScript",
-                    "QuickView",
-                    "UrlCodeHints",
-
-                    // Custom extensions we want loaded by default
-                    // "HTMLHinter",
-                    "brackets-browser-livedev",
-                    "brackets-paste-and-indent"
-                ];
+            var basePath = PathUtils.directory(window.location.href) + "extensions/default/";
+
+            // We have a set of defaults we load if not instructed to do
+            // otherwise via query string params.
+            var defaultExtensions = [
+                "CSSCodeHints",
+                "HTMLCodeHints",
+                "JavaScriptCodeHints",
+                "SVGCodeHints",
+                "HtmlEntityCodeHints",
+                "InlineColorEditor",
+                "JavaScriptQuickEdit",
+                "QuickOpenCSS",
+                "QuickOpenHTML",
+                "QuickOpenJavaScript",
+                "QuickView",
+                "UrlCodeHints",
+
+                // Custom extensions we want loaded by default
+                // "HTMLHinter",
+                "brackets-browser-livedev",
+                "brackets-paste-and-indent"
+            ];
+
+            // Add any extra extensions we found on the query string's extraExtensions param
+            if (params.get("extraExtensions")) {
+                var extraExtensions = params.get("extraExtensions").split(",");
+                extraExtensions.forEach(function (ext) {
+                    ext = ext.trim();
+                    if (defaultExtensions.indexOf(ext) === -1) {
+                        console.log('[Brackets] Loading additional extension `' + ext + '`');
+                        defaultExtensions.push(ext);
+                    }
+                });
+            }
+
+            // Disable any extensions we found on the query string's disabledExtensions param
+            if (params.get("disabledExtensions")) {
+                var disabledExtensions = params.get("disabledExtensions").split(",");
+                disabledExtensions.forEach(function (ext) {
+                    ext = ext.trim();
+                    var idx = defaultExtensions.indexOf(ext);
+                    if (idx > -1) {
+                        console.log('[Brackets] Disabling default extension `' + ext + '`');
+                        defaultExtensions.splice(idx, 1);
+                    }
```

Comment:
It's a command, you need to get the case right.  I don't think this is worth worry about, tbh.